wdqs/0.3.40/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:xenial as fetcher
+
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends unzip=6.* && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+ADD https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/0.3.40/service-0.3.40-dist.tar.gz ./service-dist.tar.gz
+
+RUN tar xfv service-dist.tar.gz && rm service-dist.tar.gz
+
+FROM openjdk:8-jdk-alpine
+
+# Blazegraph scripts require bash
+# Install gettext for envsubst command, (it needs libintl package)
+# Install curl for the loadData.sh wdqs script (if someone needs it)
+RUN set -x ; \
+    apk --no-cache add bash=\<4.5.0 gettext=\<0.19.8.2 libintl=\<0.19.8.2 curl=\<7.64.999 su-exec=\~0.2
+
+RUN addgroup -g 66 -S blazegraph
+RUN adduser -S -G blazegraph -u 666 -s /bin/bash blazegraph
+
+COPY --from=fetcher --chown=blazegraph:blazegraph /service-0.3.40 /wdqs
+
+RUN mkdir /var/log/wdqs && chown blazegraph /var/log/wdqs
```

Comment:
I added this for the logs to actually be written, however looking at what is output this might seem unnecessary unless wdqs is executed with the verbose flag.

`bash-4.4# pwd
/var/log/wdqs
bash-4.4# ls
wdqs-updater_jvm_gc.pid10.log.0.current  wdqs-updater_jvm_gc.pid12.log.0.current  wdqs-updater_jvm_gc.pid9.log.0.current
bash-4.4# tail wdqs-updater_jvm_gc.pid9.log.0.current -n 20
2020-07-08T15:30:31.643+0000: 976.667: Total time for which application threads were stopped: 0.0001903 seconds, Stopping threads took: 0.0000429 seconds
2020-07-08T15:30:31.643+0000: 976.667: Total time for which application threads were stopped: 0.0001552 seconds, Stopping threads took: 0.0000361 seconds
2020-07-08T15:30:41.646+0000: 986.670: Total time for which application threads were stopped: 0.0002666 seconds, Stopping threads took: 0.0001046 seconds
2020-07-08T15:30:41.647+0000: 986.670: Total time for which application threads were stopped: 0.0002189 seconds, Stopping threads took: 0.0000468 seconds
2020-07-08T15:30:41.647+0000: 986.671: Total time for which application threads were stopped: 0.0001524 seconds, Stopping threads took: 0.0000348 seconds
2020-07-08T15:30:51.650+0000: 996.674: Total time for which application threads were stopped: 0.0002675 seconds, Stopping threads took: 0.0001047 seconds
2020-07-08T15:30:51.651+0000: 996.674: Total time for which application threads were stopped: 0.0002464 seconds, Stopping threads took: 0.0000545 seconds
2020-07-08T15:30:51.651+0000: 996.675: Total time for which application threads were stopped: 0.0001872 seconds, Stopping threads took: 0.0000390 seconds
2020-07-08T15:31:01.654+0000: 1006.678: Total time for which application threads were stopped: 0.0001900 seconds, Stopping threads took: 0.0000392 seconds
2020-07-08T15:31:01.655+0000: 1006.678: Total time for which application threads were stopped: 0.0001305 seconds, Stopping threads took: 0.0000300 seconds
2020-07-08T15:31:01.655+0000: 1006.678: Total time for which application threads were stopped: 0.0001332 seconds, Stopping threads took: 0.0000320 seconds
2020-07-08T15:31:10.657+0000: 1015.680: Total time for which application threads were stopped: 0.0002907 seconds, Stopping threads took: 0.0001442 seconds
2020-07-08T15:31:11.658+0000: 1016.681: Total time for which application threads were stopped: 0.0002544 seconds, Stopping threads took: 0.0000524 seconds
2020-07-08T15:31:11.658+0000: 1016.682: Total time for which application threads were stopped: 0.0002219 seconds, Stopping threads took: 0.0000491 seconds
2020-07-08T15:31:11.659+0000: 1016.682: Total time for which application threads were stopped: 0.0001576 seconds, Stopping threads took: 0.0000370 seconds
2020-07-08T15:31:21.661+0000: 1026.685: Total time for which application threads were stopped: 0.0002370 seconds, Stopping threads took: 0.0000566 seconds
2020-07-08T15:31:21.662+0000: 1026.685: Total time for which application threads were stopped: 0.0001686 seconds, Stopping threads took: 0.0000403 seconds
2020-07-08T15:31:21.662+0000: 1026.686: Total time for which application threads were stopped: 0.0001558 seconds, Stopping threads took: 0.0000364 seconds
2020-07-08T15:31:31.665+0000: 1036.689: Total time for which application threads were stopped: 0.0002526 seconds, Stopping threads took: 0.0000529 seconds
2020-07-08T15:31:31.666+0000: 1036.689: Total time for which application threads were stopped: 0.0001691 seconds, Stopping threads took: 0.0000402 seconds
bash-4.4# 
`

Reply:
+1 for including a prepared directory in the image if it would be used with the verbose flag!